front/app/containers/UsersEditPage/ProfileForm.tsx

```diff
@@ -328,6 +328,11 @@ class ProfileForm extends PureComponent<Props, State> {
           />
 
           <SectionField>
+            <FormLabel
+              hidden
```

Comment:
Why is this label hidden?

Reply:
We should show it imo. (Would also delete the subheading under 'Your account information'.)

![ava](https://user-images.githubusercontent.com/16427929/153144415-c847d71d-f7bb-4a8d-a2ca-512c7342510d.png)
'

Reply:
Not sure, actually, this is how it looked originally, so I hid it. But I agree with you - will show it 👍